README.md

```diff
@@ -2,14 +2,90 @@
 
 # Wikipedia KaiOS 
 
-The official Wikipedia KaiOS app.
+The official Wikipedia KaiOS app. [KaiOS](https://developer.kaiostech.com/) is a web-based mobile operating system that enables a new category of smart feature phones, a successor of the discontinued Firefox OS.
 
-Tasks and issues are tracked on [Phabricator](https://phabricator.wikimedia.org/project/profile/4305/).
+KaiOS applications are similar to Progressive Web Apps (PWAs):
+* Pure web applications
+* Manifest file
+* Installed on the device
+* Have access to the required APIs for notifications, offline, storage, audio, etc
+
+They are also different:
+* Format of the manifest is different
+* Size of the icons is specific to KaiOS
+* Need to be designed for a small, non-touch screen with a few physical buttons
+* Several APIs are specific to the platform (not web standards)
+* Running performance on the device
+
+The Wikipedia KaiOS app is built with [Preact](https://preactjs.com/) and we use [Cypress](https://www.cypress.io/) for testing. Tasks and issues are tracked on [Phabricator](https://phabricator.wikimedia.org/project/profile/4305/).
+
+## Installation requirements
+
+* Make sure git is installed: `brew install git`
+* Make sure node is installed: `node -v` or `brew install node`
+* Make sure npm is installed: `npm -v` or `brew install npm`
+
+## To run this app in your browser
+
+### Clone the app (only the first time)
+
+1. Create a directory for code: `mkdir ~/code && cd ~/code`
+2. Download the app source repository: `git clone https://github.com/wikimedia/wikipedia-kaios.git`
+
+### Open the app on your computer
+1. Make sure you are in the source directory: `cd wikipedia-kaios`
+2. Downloading the latest code: `git pull`
+3. Install app dependencies: `npm install`
+4. Build the app: `npm run dev`
+
+Now the app is running on your computer at http://127.0.0.1:8080  on your browser
+
+## To run this app on a device
+
+### Clone the app (if you haven't done this alredy)
+
+1. Create a directory for code: `mkdir ~/code && cd ~/code`
+2. Download the app source repository: `git clone https://github.com/wikimedia/wikipedia-kaios.git`
+3. Go to the app source: `cd wikipedia-kaios`
+
+### Build the app
+1. Install app dependencies: `npm install`
+2. Build the app: `npm run build`
+
+### Connect your device
+1. Connect your device to your computer using a USB cable
+2. Enable debug mode on the device by typing: `*#*#33284#*#*`
+3. Check that adb is running and sees your device: `adb devices`
+
+_Steps 2, 3 and 4 need to be redone when you disconnect and reconnect your device or restart your computer._
```

Comment:
this is not true for my device, not sure this should be here at all

Reply:
this line is a `nice to have` note, let's delete this line to keep our readme in a minimum step

Reply:
Are you talking about lines 57-60 or only line 60?

Reply:
only 60